README.md

```diff
@@ -38,3 +38,19 @@ poetry run doc8 -q docs
 ```
 
 It's OK if some tests are skipped.
+
+
+## Configuration file
```

Comment:
We also need to support `cli` usage. See https://github.com/zheller/flake8-quotes/blob/master/flake8_quotes/__init__.py#L99